service-registry/src/test/java/org/apache/servicecomb/serviceregistry/task/TestMicroserviceRegisterTask.java

```diff
@@ -498,12 +508,36 @@ public void testLocalSchemaAndServiceCenterSchemaDiff(@Mocked ServiceRegistryCli
         result = otherMicroservice;
         srClient.getSchemas(anyString);
         result = onlineSchemasHolder;
+        srClient.getSchema(anyString, anyString);
+        result = "abcdefghijklmn";
       }
     };
 
-    microservice.addSchema("s1", "abdc");
+    microservice.addSchema("s1", "abcdefghijklmnopqrstuvwxyz");
     microservice.setEnvironment("prod");
     MicroserviceRegisterTask registerTask = new MicroserviceRegisterTask(eventBus, srClient, microservice);
-    registerTask.run();
+
+    Logger logger = Logger.getLogger(MicroserviceRegisterTask.class);
```

Comment:
use org.apache.servicecomb.foundation.test.scaffolding.log.LogCollector
after finished, do not forget invoke logCollector.teardown()